### ISSUE TITLE
Transform cx arguments into string concats

### ISSUE THREAD

**Issue by iiegor** [iiegor/react-experiments#1]
Transform `cx('Foo')` to `Foo` and `cx('Foo', 'Bar')` to `Foo Bar` to avoid possible allocations.

_Original Home.react.js_

``` javascript
import React from 'react';

const cx = require('cx');

class Home extends React.Component {
  render() {
    return (
      <div className={cx('Home/root', 'Home/anotherRule')}>Hello world!</div>    
    );
  }
}
```

_Expected Home.react.js after compilation_

``` javascript
import React from 'react';

const cx = require('cx');

class Home extends React.Component {
  render() {
    return (
      <div className="Home-root Home-anotherRule">Hello world!</div>    
    );
  }
}
```

See [scripts/babel/cx-replacement.js](https://github.com/iiegor/react-experiments/blob/master/scripts/babel/cx-replacement.js).

**Comment by iiegor** [comment]
An experimental version of this was landed in b0785aa775c0a192aca5068f64db878401c99345.

**Comment by iiegor** [comment]
Almost completed in e1faf32357c3137872f939c13237c45287770ff3.

_Before compilation:_
![img](http://i.imgur.com/AAEzSjH.png)

_After compilation:_
![img](http://i.imgur.com/8S6lVtQ.png)
